yaml/builders/quay-hub-update.yaml

```diff
@@ -0,0 +1,48 @@
+# Requires:
+# {targetOS}
+# {gitproject}
+# {gituser}
+- builder:
+    name: 'quay-hub-update'
+    parameters:
+        - string:
+              name: description
+              default: ''
+              description: ""
+
+    builders:
+        - shell: |
+            #!/bin/bash
+            set -ex
+            envsubst > upload_quay.sh << 'EOF'
+            echo -n '{{"description":"' > data.json
+            echo -n $2 >> data.json
+            echo -n '"}}' >> data.json
+            cat data.json
```

Comment:
I expect these are debugging outputs, are they kept here on purpose? Same for `cat upload_quay.sh` later.